src/ontology/eupath_dev.owl

```diff
@@ -16452,6 +16452,90 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043212 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043212">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000047"/>
+        <obo:IAO_0000115>A count of the number of copies of the interspecific chromosomal inversion (2La) in Anopheles gambiae complex.</obo:IAO_0000115>
+        <obo:IAO_0000116>2La is located on chromosome arm 2L:20524058-42165532 (approximate location in the AgamP3 assembly; GenBank: GCF_000005575.2) as described in: Sharakhov, I.V., White, B.J., Sharakhova, M.V., Kayondo, J., Lobo, N.F., Santolamazza, F., Della Torre, A., Simard, F., Collins, F.H. and Besansky, N.J., 2006. Breakpoint structure reveals the unique origin of an interspecific chromosomal inversion (2La) in the Anopheles gambiae complex. Proceedings of the National Academy of Sciences, 103(16), pp.6258-6262. PMID: 16606844</obo:IAO_0000116>
+        <obo:IAO_0000233>https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues/117</obo:IAO_0000233>
+        <rdfs:label>number of 2La Anopheles gambiae complex chromosomal inversion</rdfs:label>
```

Comment:
It's tricky without adding a lot more words:

Number of copies of the 2La Anopheles gambiae complex chromosomal inversion

Reply:
Agree with Bob's suggested form of label.